antora.yml

```diff
@@ -14,104 +14,189 @@ asciidoc:
     openshift_application_data_services_apache_kafka_url: https://console.redhat.com/application-services/streams/kafkas
     camel_url: https://camel.apache.org/
+    visual_studio_code_url: https://code.visualstudio.com/
+    visual_studio_code_swf_extension_url: https://marketplace.visualstudio.com/items?itemName=kie-group.swf-vscode-extension
+    k9s_url: https://k9scli.io/
+    graalvm_url: https://www.graalvm.org/
+    graalvm_native_image_url: https://www.graalvm.org/22.0/reference-manual/native-image/
+    slf4j_simple_maven_repo_url: https://mvnrepository.com/artifact/org.slf4j/slf4j-simple
     # must align this version
     camel_extensions_url: https://camel.apache.org/camel-quarkus/3.2.x/reference/extensions
     kaoto_url: https://marketplace.visualstudio.com/items?itemName=redhat.vscode-kaoto
     minikube_url: https://minikube.sigs.k8s.io
-    kogito_serverless_operator_url: https://github.com/kiegroup/kogito-serverless-operator/tree/9.99.x-prod
-    docs_issues_url: https://github.com/apache/incubator-kie-kogito-docs/issues/new
-    # xreferences to documents within the serverless-logic documentation
+    minikube_start_url: https://minikube.sigs.k8s.io/docs/start/
+    kind_install_url: https://kind.sigs.k8s.io/docs/user/quick-start/#installation
+    kogito_serverless_operator_url: https://github.com/kiegroup/kogito-serverless-operator/
+    docs_issues_url: https://github.com/kiegroup/kogito-docs/issues/new
+    ocp_local_url: https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.17
+    ocp_knative_serving_install_url: https://docs.openshift.com/container-platform/4.12/serverless/install/installing-knative-serving.html
+    ocp_knative_eventing_install_url: https://docs.openshift.com/container-platform/4.12/serverless/install/installing-knative-eventing.html
+    ocp_kn_cli_url: https://docs.openshift.com/container-platform/4.12/serverless/install/installing-kn.html
+    k8n_secrets_url: https://kubernetes.io/docs/concepts/configuration/secret
+
+    #
+    # xreferences to documents within the documentation
+    #
     data_index_xref: xref:data-index/data-index-core-concepts.adoc
     job_service_xref: xref:job-services/core-concepts.adoc
-    # string unication references withing serverless logic documentation
-    data_index_ref: Data Index
-    workflow_instance: workflow instance
-    workflow_instances: workflow instances
-    sonataflow_devmode_devui_url: /q/dev/org.kie.kogito.kogito-quarkus-serverless-workflow-devui/
+
+    # Tag to use with community-only images
+    sonataflow_non_productized_image_tag: NO_TAG
+    operator_community_prod_yaml: https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/9.100.x-prod/operator.yaml
+    operator_community_prod_root: https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/9.100.x-prod
+    sonataflow_operator_imagename: registry.redhat.io/openshift-serverless-1/logic-rhel8-operator
```

Comment:
Same here

Reply:
Yea,  I see your point. This is however using same rules as other imagename URLs, I can unify across all repos  later right now this should stay as is imho